pymodels/TB_V04_01/lattice.py

```diff
@@ -150,7 +150,11 @@ def create_lattice(optics_mode=default_optics_mode):
     sector04 = [s04_1, qf4, s04_2, qd4, s04_3, septin]
 
     # TB beamline
-    ltlb = [inicio, sector00, sector01, sector02, sector03, sector04, fim]
+    if add_li_triplets:
+        ltlb = [
+            inicio, sector00, sector01, sector02, sector03, sector04, fim]
```

Comment:
@xresende , could you standardize the use of english here?

Reply:
done